Cargo.toml

```diff
@@ -59,6 +60,7 @@ default = ["serde", "std"]
 legacy-ops = []
 maths = []
 maths-nopanic = ["maths"]
+rkyv = ["rkyv/size_32"]
```

Comment:
```suggestion
rkyv = ["rkyv/size_32", "rkyv/std", "std"]
```

Reply:
rkyv can be used without std. I added rkyv/std in the std feature of rust_decimal

Reply:
Oh I just say your last comment.